cloudformation_seed/cfn_stackset.py

```diff
@@ -293,12 +332,22 @@ def update_stackset(self) -> None:
         log.debug(' Parameters '.center(48, '-'))
         log.debug(p)
         log.debug('-'.center(48, '-'))
-        params = {
-            'StackSetName': self.stack_name,
-            'TemplateURL': self.template.template_url,
-            'Parameters': p,
-            'Capabilities': self.caps,
-        }
+        if self.stack_tags:
+            params = {
+                'StackSetName': self.stack_name,
+                'TemplateURL': self.template.template_url,
+                'Parameters': p,
+                'Capabilities': self.caps,
+                'Tags': self.formatted_stack_tags
+            }
+        else:
+            params = {
+                'StackSetName': self.stack_name,
+                'TemplateURL': self.template.template_url,
+                'Parameters': p,
+                'Capabilities': self.caps,
+                'Tags': []
```

Comment:
Same as before. If `self.stack_tags` is set to `[]` by default, there's no need for this condition.